src/main/java/com/inqoo/trainingservice/app/controller/CategoryController.java

```diff
@@ -0,0 +1,27 @@
+package com.inqoo.trainingservice.app.controller;
+
+import com.inqoo.trainingservice.app.models.Category;
+import com.inqoo.trainingservice.app.service.CategoryService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/main")
+public class CategoryController {
+    public CategoryController(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
+
+    private CategoryService categoryService;
```

Comment:
Fields should be declared above constructor

Reply:
Done.